src/fileio/fileio.cpp

```diff
@@ -120,7 +126,8 @@ fs::path get_posix_tempdir() {
 void check_for_intel() {
 	union {uint16_t x; uint8_t c;} endian;
 	endian.x = 1;
-	mac_is_intel = endian.c;
+	_mac_is_intel = endian.c;
+	_checked_for_intel = true;
```

Comment:
Is this function still called somewhere? If not, I think its contents should just be merged into the `mac_is_intel()` function. Then these two variables can just become static local variables.